app/models/api_search.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module ApiSearch
+  def initialize(options = {})
+    super
+
+    @highlight_options[:pre_tags]  ||= Api::V2::HighlightOptions::DEFAULT[:pre_tags]
+    @highlight_options[:post_tags] ||= Api::V2::HighlightOptions::DEFAULT[:post_tags]
```

Comment:
Not sure if we will be using it but could it be beneficial to set `attrs` here for the instantiated variables.

Reply:
I not sure why we would need to set attrs. It everything works without them I rather keep it like it is now.